src/backend/el.d

```diff
@@ -65,22 +65,24 @@ struct elem
     enum IDelem = 0x4C45;   // 'EL'
     //elem_debug(e) assert((e)->id == IDelem)
 
+    version (OSX) // workaround https://issues.dlang.org/show_bug.cgi?id=16466
+        align(16) eve EV; // variants for each type of elem
+    else
+        eve EV;           // variants for each type of elem
+
```

Comment:
The `debug short` is at the beginning for a good reason. First off, it doesn't affect release code, so the size doesn't matter. The reason it is first is because freeing a pointer almost always overwrites the first word in the data, so putting it first makes it far more likely to catch dangling pointer bugs (which is why it is there).